main.py

```
@@ -462,16 +459,16 @@ async def delete_like(
 @app.delete("/comments/{comment_id}", status_code=status.HTTP_204_NO_CONTENT)
 async def delete_comment(
         comment_id: Annotated[str, Path(description="삭제할 댓글 ID")],
```

Ruff (0.14.13)

Unused function argument: `comment_id`

(ARG001)